gray_model.py

```diff
@@ -208,5 +209,8 @@ def forward(self, x):
         out= self.subpixel(out)
         out = self.conv_output(out)
         out = torch.add(out, residual)
+        noise_std = torch.reshape(noise_std, (residual.size()[0], 1, 1, 1))
+        if self.exact_recon:
+            out = torch.subtract(residual, noise_std * out)
 
-        return out
\ No newline at end of file
+        return out
```

Comment:
I can't make this "fake change" disappear... I even tried to do it by hand in github but it didn't work.

Reply:
sure no prob